Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.16.3-alpine3.12 as builder
```

Comment:
Can we specify somewhere that this Dockerfile is janitor-only? Or call it Dockerfile-janitor? Or put it in a janitor directory?

Just something to avoid someone confusing this be an okctl-Dockerimage somehow.

Reply:
I will add a comment, github action requires it be called Dockerfile, and Docker needs the repo as context